kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/config/ExecutorConfiguration.java

```diff
@@ -63,6 +75,18 @@ public String getExecutorUri() {
         return executorUri;
     }
 
+    public String getCommand() {
```

Comment:
There's inconsistent use of the `@JsonProperty` annotation in this file.  It should be made consistent.  In particular `getNetworkMode` and `getCniNetwork` would turn out camelcase in JSON which is bad.  Should be snake_case.